WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignsViewController.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+final class BlazeCampaignsViewController: UIViewController {
+
+    // MARK: - Views
+
+    private lazy var dismissButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(image: UIImage(systemName: "plus"),
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(promotePostButtonTapped))
+        return button
+    }()
+
+    // MARK: - Properties
+
+    private var blog: Blog
+
+    // MARK: - Initializers
+
+    init(blog: Blog) {
+        self.blog = blog
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        // This VC is designed to be initialized programmatically.
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupNavBar()
+    }
+
+    // MARK: - Private helpers
+
+    private func setupView() {
+        view.backgroundColor = .DS.Background.primary
+    }
+
+    private func setupNavBar() {
+        title = Strings.navigationTitle
+        navigationItem.rightBarButtonItem = dismissButton
+    }
+
+    @objc private func promotePostButtonTapped() {
```

Comment:
[Nit] I suggest renaming this to `plusButtonTapped` cause we're going to have another button in the view likely named `promoteButton`

Reply:
Sounds good. I'll address this in https://github.com/wordpress-mobile/WordPress-iOS/pull/20814 to avoid conflicts